presto-parser/src/main/antlr4/io/prestosql/sql/parser/SqlBase.g4

```diff
@@ -64,9 +64,17 @@ statement
     | ALTER TABLE tableName=qualifiedName
         ADD COLUMN column=columnDefinition                             #addColumn
     | ANALYZE qualifiedName (WITH properties)?                         #analyze
+    | CREATE (OR REPLACE)?  MATERIALIZED VIEW
+        (IF NOT EXISTS)?
+        qualifiedName
+        (SECURITY (DEFINER | INVOKER))?
```

Comment:
SECURITY INVOKER doesn't make sense for materialized views. They need to run with DEFINER mode always, so let's just get rid of this

Reply:
Done. (I had done it as a part of 'additional syntax' commit, but that commit is going away, so using this commit to remove the 'security' mode code).